testutils/fakeflags/logflags.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2015 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License. See the AUTHORS file
+// for names of contributors.
+//
+// Author: Tobias Schottdorf
+
+package fakeflags
+
+import "github.com/cockroachdb/cockroach/util/log/logflags"
+
+type fakeValue struct{}
+
+func (*fakeValue) String() string     { return "" }
+func (*fakeValue) Set(_ string) error { return nil }
```

Comment:
no, `string` doesn't implement anything and the `flag` package doesn't expose its internal type for this.